app/src/main/java/com/space/quizapp/presentation/ui/home/custom_view/GpaVectorView.kt

```diff
@@ -0,0 +1,78 @@
+package com.space.quizapp.presentation.ui.home.custom_view
+
+import android.content.Context
+import android.graphics.*
+import android.util.AttributeSet
+import android.view.View
+import androidx.core.content.ContextCompat
+import androidx.core.graphics.PathParser
+import com.space.quizapp.R
+
+/**
+ * A custom view that draws a blue vector with with white arrow and light blue space for gpa.
+ */
+class GpaVectorView(context: Context, attrs: AttributeSet?) : View(context, attrs) {
```

Comment:
ლუკა, ეს არ იყო საჭირო რომ დაგეხატა ;დ
მარტო ეს ვიუები:
https://www.figma.com/file/UgTXNAXH2ScX6OHRREkSIZ/Designs-for-interns?type=design&node-id=261-1819&t=OkhyVnDgRbOuWmvU-0

https://www.figma.com/file/UgTXNAXH2ScX6OHRREkSIZ/Designs-for-interns?type=design&node-id=462-1863&t=OkhyVnDgRbOuWmvU-0

დანარჩენებისთვის შეგიძლია drawableები შექმნა

Reply:
changed in a later pull request